### PR TITLE
Use local fonts in BO default theme

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The default theme loaded its fonts from the Google fonts API, which obviously doesn't work when offline. This PR includes those fonts for legacy pages in the BO. A follow-up PR should fix this on migrated pages as well.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Partial fix for #12772
| How to test?  | Load a legacy page while disconnected from the internet, see that the fonts are correct.

This PR also adds the webpack cleanup plugin on the default theme. Some assets were removed (see diff to learn which ones), which would indicate that they are no longer used. We should verify that they are indeed no longer used.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16209)

### CODE REVIEW

File: admin-dev/themes/default/sass/modules/_variables.sass [hunk 1]
```diff
@@ -72,9 +72,8 @@ $body-bg: $bg-content-color
 $text-color: $gray
 
 // Typography
-$url-font-content: "//fonts.googleapis.com/css?family=Open+Sans:400,400i,600,700" !default
+$FontPathOpenSans: '../node_modules/open-sans-fonts/open-sans'
```
Comment (Progi1984) [inline admin-dev/themes/default/sass/modules/_variables.sass:75]:
```suggestion
$fontPathOpenSans: '../node_modules/open-sans-fonts/open-sans'
```

Comment (eternoendless) [reply admin-dev/themes/default/sass/modules/_variables.sass:75]: Unfortunately this variable must be defined exactly like that in order for the required library to work.

Comment (Progi1984) [reply admin-dev/themes/default/sass/modules/_variables.sass:75]: Ok. Sorry :pray:

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by eternoendless** [comment]
Be aware that because the new theme is still loading fonts from Google, when switching from legacy pages to new pages fonts will have to be downloaded again by the client.

**Comment by PierreRambaud** [comment]
Thanks @eternoendless

**Pull request merged**